CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+[authState]: https://github.com/okta/okta-auth-js#authstatemanager
+
+# 3.0.0
+
+### Breaking Changes
+
+- [#27](https://github.com/okta/okta-vue/pull/27) See [MIGRATING](MIGRATING.md) for detailed information.
```

Comment:
Are we requiring them to install okta-auth-js themselves?  That's a breaking change I'd mention.

Reply:
Within this PR, okta-vue still installs `okta-auth-js` for the users. I am playing with `vue-cli-service` to build the lib, which can make `okta-auth-js` as a peer dependency (probably in separate PR)